src/honey_badger/epoch_state.rs

```diff
@@ -376,3 +382,17 @@ where
         }
     }
 }
+
+/// A session identifier for a `Subset` sub-algorithm run within an epoch. It consists of the epoch
+/// number, and an optional `HoneyBadger` session identifier.
```

Comment:
Did our abstraction leak here again? =)

Reply:
This is inside the `honey_badger` module, so I think it's fine to refer to its session identifier?